test/stress/cluster_stress_test.go

```diff
@@ -0,0 +1,166 @@
+// +build stress
+
+package stress
+
+import (
+	"context"
+	"math/rand"
+	"strconv"
+	gosync "sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/yorkie-team/yorkie/client"
+	"github.com/yorkie-team/yorkie/pkg/document"
+	"github.com/yorkie-team/yorkie/pkg/document/proxy"
+	"github.com/yorkie-team/yorkie/pkg/types"
+	"github.com/yorkie-team/yorkie/test/helper"
+	"github.com/yorkie-team/yorkie/yorkie"
+	"github.com/yorkie-team/yorkie/yorkie/backend/sync"
+	"github.com/yorkie-team/yorkie/yorkie/backend/sync/memory"
+)
+
+func TestClusterStress(t *testing.T) {
+	t.Run("watch document test", func(t *testing.T) {
```

Comment:
This is a test written to ensure that documents are safely synchronized when multiple agents are activated and requests are randomly sent to each agent.

Reply:
Why not just be in the integration test?

Reply:
I wanted to see what happens when I forward a lot of requests to multiple servers. For example, I thought it would be good to check unexpected situations such as unexpected errors and performance issues. Do you think it is appropriate to move to integration testing?
Come to think of it, it seems that this type of test is also necessary for integration tests.

Reply:
I think this test is complicated. Let's start with the integration test by simplifying the test.